README.md

```diff
@@ -1,75 +1,13 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] README file should be descriptive and includes all the necessary headers and sections, kindly consider using [The README](https://github.com/microverseinc/readme-template) provided by microverse